runtime/doc/options.txt

```diff
@@ -4872,8 +4741,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	languages, no matter what you set 'mkspellmem' to.
 
 				   *'modeline'* *'ml'* *'nomodeline'* *'noml'*
-'modeline' 'ml'		boolean	(Vim default: on (off for root),
-				 Vi default: off)
+'modeline' 'ml'		boolean	(default: on (off for root))
```

Comment:
These types of "Vi default ..." notes are still relevant because we support the `set foo&vi` invocation. E.g.:

    set modeline&vi

Will set `modeline` to "off". 

And I see no reason to remove support for `&vi` because it doesn't add complexity, it's basically just a set of defaults listed in the `option.c:vimoption_T` table.